### PR TITLE
Update navigation

### LINKED ISSUE #60: Update navigation & move store page to /collections

### PR DESCRIPTION
**Pull request by solazio**
## Done

- Update the navigation
- Move store page to `/collections`
- Add `/about` page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8045
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- [List additional steps to QA the new features or prove the bug has been resolved]

## Issue / Card

Fixes #60

### CODE REVIEW

File: static/sass/styles.scss [hunk 1]
```diff
@@ -32,6 +32,7 @@ $theme-default-nav: dark;
 @include vf-p-media-object;
 @include vf-p-muted-heading;
 @include vf-p-navigation;
+@include vf-p-search-box;
```
Comment (Lukewh) [inline static/sass/styles.scss:35]:
```suggestion

```

File: templates/about.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    </p>
+    <h2>What is dev-sec-ops?</h2>
+    <p>
+      Dev-sec-ops is the merging of product development, security, and operations. Practitioners of dev-sec-ops encourage the formation of multi-disciplinary teams that consider the operational and security aspects of enterprise software at the same time, ideally during product development itself. Instead of staying silent, or simply making recommendations about security and operations, dev-sec-ops teams aim to deliver automated solutions that cover all these elements together.
+    </p>
+    <p>
+      Operators are dev-sec-ops because they are in effect total automation - all of the product installation, integration, operations and lifecycle management are encapsulated in the code of the operator, so all of the security and operations elements are by definition included in operator design and implementation.
```
Comment (anthonydillon) [inline templates/about.html:39]: Can you make this dash an `&mdash;`

File: templates/collections.html [hunk 3]
```diff
@@ -0,0 +1,48 @@
+{% extends 'base_layout.html' %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1WcNN2RlQQrbLSP5ZgcIsAeQAHj5h_yDW6Yd5oBMmU3M{% endblock meta_copydoc %}
```
Comment (anthonydillon) [inline templates/collections.html:3]: This looks like the same copy doc as the one above

File: templates/about.html [hunk 4]
```diff
@@ -0,0 +1,61 @@
+{% extends 'base_layout.html' %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1WcNN2RlQQrbLSP5ZgcIsAeQAHj5h_yDW6Yd5oBMmU3M{% endblock meta_copydoc %}
+
+{% block meta_description %}Charmhub description{% endblock %}
```
Comment (anthonydillon) [inline templates/about.html:5]: Can you take a sentence from the first paragraph that would be better then this.

File: templates/about.html [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+{% block content %}
+<section class="p-strip--charmhub is-shallow is-dark">
+  <div class="u-fixed-width">
+    <h1 class="p-heading--2">This is the about page</h1>
```
Comment (anthonydillon) [inline templates/about.html:10]: Surely "About Open Operators Collection" is better?

Comment (solazio) [reply templates/about.html:10]: @anthonydillon this about page is more a placeholder for now. Daniel is looking at it and will properly implement it after he's done.

**Pull request merged**